ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+clang-format -style=file -i stl/inc/* stl/inc/cvt/* stl/inc/experimental/* stl/src/* 2>&1

```

Comment:
```suggestion
clang-format --style=file -i stl/inc/* stl/inc/cvt/* stl/inc/experimental/* stl/src/* 2>&1
```

Reply:
https://clang.llvm.org/docs/ClangFormat.html disagrees, and our internal batch file has been using the `-style=file` option successfully.

Reply:
![image](https://user-images.githubusercontent.com/1544943/65562389-1ad78400-defb-11e9-8f94-1b0e80f96e43.png)

Reply:
Yep - withdrawn.